piff/basis_interp.py

```diff
@@ -24,6 +24,48 @@
 from .interp import Interp
 from .star import Star
 
+import jax
+# If uncommented, the following line will make the code run in double precision
+# and have identical results as running the original code in double precision.
+# jax.config.update("jax_enable_x64", True)
+from jax import jit
+from jax import numpy as jnp
+from jax import vmap
+
+
+# Bellow are implementations of _solve_direct using JAX.
+# if jax.config.update("jax_enable_x64", True) it will give the
+# same results as the original code in double precision, but will run
+# slower, but still faster than the numpy/scipy version.
+
+@jit
+def jax_solve(ATA, ATb):
+    # Original code:
+    # dq = scipy.linalg.solve(ATA, ATb, assume_a='pos', check_finite=False)
+    # New code:
+    factor = (jax.scipy.linalg.cholesky(ATA, overwrite_a=True, lower=False), False)
```

Comment:
This is mostly jax's poor API here, but that bare `False` is bad.  It should be named `lower` to be self-documenting.